code/game/data_huds.dm

```diff
@@ -554,3 +557,13 @@ Diagnostic HUDs!
 	var/image/holder = hud_list[DIAG_AIRLOCK_HUD]
 	holder.icon_state = "electrified"
 	set_hud_image_active(DIAG_AIRLOCK_HUD)
+
+/*~~~~~~~~~~~~~~~~~
+	Hacked APCs!
+~~~~~~~~~~~~~~~~~~*/
+/obj/machinery/power/apc/proc/set_hacked_hud()
+	var/image/holder = hud_list[MALF_APC_HUD]
+	holder.loc = get_turf(src)
+	holder.plane = HUD_PLANE
```

Comment:
I think you're supposed to use one of the `SET_PLANE`s for this, but also `HUD_PLANE` is for screen elements like inventory, so I don't think it's supposed to be used for this